front/src/components/pages/ChannelSettingPage/ChannelSettingPage.tsx

```diff
@@ -0,0 +1,153 @@
+  },
+  form: {
+    paddingTop: '10%',
+    height: '100%',
+    width: '100%',
+  },
+}));
+
+const ChannelSettingPage = (): React.ReactElement => {
+  const classes = useStyles();
+  const history = useHistory();
+  const { user } = usePongState();
+  const [channel, setChannel] = useState<ChannelType>();
+  const [type, setType] = useState('user');
+
+  const menuItems = ['유저 정보 변경', '채널 정보 변경'];
+
+  const onMenuChanged = (target: string) => {
+    const typeByMenu: Record<string, 'user' | 'channel'> = {
+      '유저 정보 변경': 'user',
+      '채널 정보 변경': 'channel',
+    };
+    setType(typeByMenu[target]);
+  };
+
+  // TODO channel-id에 따라 받아오는 데이터가 달라야 함.
+
+  useEffect(() => {
+    const getChannelInfo = async () => {
+      // await axios.get(`${process.env.REACT_APP_API_URL}/channel/${id}`);
+      const getInfo = ():
+          // TODO 백엔드에서 채널 데이터 완성시
+          // 아래의 내용을 변경해야 함.
+            Promise<ChannelType> => new Promise<ChannelType>((resolve) => {
+              setTimeout(() => resolve({
+                id: 1,
+                isPrivate: true,
+                name: '신기한 채널 이름',
+                joinCnt: 3,
+                totalCnt: 10,
+                owner: 1,
+                member: [
+                  {
+                    id: 10,
+                    image: 'https://cdn.intra.42.fr/users/medium_sunpark.jpg',
+                    name: 'kmin1',
+                    email: 'kmin@test.com',
+                    online: true,
+                    score: 123,
+                    gameHistory: {
+                      win: 12,
+                      lose: 10,
+                    },
+                  },
+                  {
+                    id: 20,
+                    image: 'https://cdn.intra.42.fr/users/medium_sunpark.jpg',
+                    name: 'kmin2',
+                    email: 'kmin@test.com',
+                    online: true,
+                    score: 123,
+                    gameHistory: {
+                      win: 12,
+                      lose: 10,
+                    },
+                  },
+                  {
+                    id: 30,
+                    image: 'https://cdn.intra.42.fr/users/medium_sunpark.jpg',
+                    name: 'kmin3',
+                    email: 'kmin@test.com',
+                    online: true,
+                    score: 123,
+                    gameHistory: {
+                      win: 12,
+                      lose: 10,
+                    },
+                  },
+                ],
+                adminUserIds: [50, 30],
+              }), 1000);
+            });
+      setChannel(await getInfo());
+    };
+
+    getChannelInfo();
+  }, []);
+
+  console.log(channel, channel?.owner, user.id);
+
+  if (channel
+    && (channel.owner !== user.id && !channel.adminUserIds.includes(user.id))) {
+    history.push('/channel');
+  }
+
+  return (
+    <Grid className={classes.root} container>
+      <Typo variant="h3">{channel?.name}</Typo>
```

Comment:
```suggestion
      <Typo variant="h5">{channel?.name}</Typo>
```

채널 이름 표시를 좀 더 작게 해주세요.